plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduPageSink.java

```diff
@@ -215,6 +216,13 @@ public void storeMergedRows(Page page)
             for (int position = 0; position < page.getPositionCount(); position++) {
                 long operation = TINYINT.getLong(operationBlock, position);
 
+                checkState(operation <= UPDATE_OPERATION_NUMBER,
```

Comment:
Treat them as enums. The operations are numbers for technical purposes, but their ordering carries no meaning.
Thus they can only be compared for equality, but should never be compared with less/greater than.